voorhoede/heads-up
# Feat/remove google tab

I've removed the Google tab and related files.

## What change(s) were made
- Removed Google rich previews.
- Removed `google.view.vue` view.
  - Removed item from sidebar in the web app and extension.
- Removed related utility functions.

## How to test
- Run web app and/or Chrome extension locally and check if the Google tab in the sidebar is gone and no errors/warnings related to these changes are shown in the console.

## Related Trello ticket(s)
- [https://trello.com/c/jbOJLnLc](https://trello.com/c/jbOJLnLc)

## Checks
- [x] Checked web app.
